charts/scalar-admin-for-kubernetes/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: scalar-admin-for-kubernetes
 description: Scalar Admin for Kubernetes
 type: application
-version: 1.0.0-SNAPSHOT
-appVersion: 1.2.0
+version: 2.0.0-SNAPSHOT
+appVersion: 2.0.0-SNAPSHOT
```

Comment:
Scalar Admin for Kubernetes has the `SNAPSHOT` version. So, I use the `SNAPSHOT` version for `appVersion` in the `main` branch.
https://github.com/scalar-labs/scalar-admin-for-kubernetes/pkgs/container/scalar-admin-for-kubernetes/166941489?tag=2.0.0-SNAPSHOT